pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -2489,6 +2436,18 @@ func (fs *eosfs) getRootAuth(ctx context.Context) (eosclient.Authorization, erro
 	return eosclient.Authorization{Role: eosclient.Role{UID: "0", GID: "0"}}, nil
 }
 
+func (fs *eosfs) getDaemonAuth(ctx context.Context) (eosclient.Authorization, error) {
```

Comment:
Can you put some comments here?

Reply:
Explaining what this force single user does

Reply:
re the force single user, this is documented on the field in the config (so you see it when you hover your mouse). Will put in a line explaining what the daemon auth is used for!